File: SimpleFragment/app/src/main/java/com/example/simplefragment/MainActivity.java
package com.example.simplefragment;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Menu;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    MainFragment mainFragment;
    MenuFragment menuFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mainFragment = (MainFragment) getSupportFragmentManager().findFragmentById(R.id.mainFragment);
        menuFragment = new MenuFragment();
    }
        public void onFragmentChanged(int index)
        {
            if(index == 0)
            {
                getSupportFragmentManager().beginTransaction().replace(R.id.container,menuFragment).commit();
                Toast.makeText(getApplicationContext(),"Go to Menu",Toast.LENGTH_SHORT).show();
            }
            else if(index == 1)
            {
                getSupportFragmentManager().beginTransaction().replace(R.id.container,mainFragment).commit();
                Toast.makeText(getApplicationContext(),"Return to Main",Toast.LENGTH_SHORT).show();
            }
        }

}

File: MyFirstKotlinApp/app/src/main/java/com/example/myfirstkotlinapp/MainActivity.kt
package com.example.myfirstkotlinapp

import android.content.Intent
import android.graphics.Color
import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.text.Editable
import android.text.TextWatcher
import android.view.View
import android.widget.Toast
import com.example.myfirstkotlinapp.databinding.ActivityMainBinding

private lateinit var binding:ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        var count = 0
        binding.textView1.text = count.toString()
        binding.textView1.setTextSize(50F)

        binding.clickBtn.setOnClickListener{
            count++
            binding.textView1.text = count.toString()
            when(count){
                10 -> {
                    Toast.makeText(this@MainActivity,"10 click",Toast.LENGTH_SHORT).show()
                    binding.textView1.setTextColor(Color.BLUE)
                }
                20 -> {
                    Toast.makeText(this@MainActivity,"20 click",Toast.LENGTH_SHORT).show()
                    binding.textView1.setTextColor(Color.MAGENTA)
                }
                else ->{
                    binding.textView1.setTextColor(Color.BLACK)
                }
            }
        }
    }


}